backend/src/routes/user/index.ts

```diff
@@ -3,13 +3,15 @@ import {
   updateUserByIdInDB,
   getUserById,
   getAllUsers,
+  getUserNamesByUids,
 } from '../../controllers/user';
 import express, { Router } from 'express';
 
 const userRoute: Router = express.Router();
 
 userRoute.get('/:uid', getUserById);
 userRoute.get('/', getAllUsers);
+userRoute.post('/names', getUserNamesByUids);
```

Comment:
you probably dont want post here?

Reply:
I have initially tried get, but can't seem to figure out how to send the array in get request. Axios does not support data object in the body for get requests, but I also tried putting it in params object, as well as directly as a string in the url params, but whatever I do, best case scenario is that I get an empty response data array